tests/end-to-end/package-lock.json

```diff
@@ -0,0 +1,8630 @@
+{
```

Comment:
Is this file intended to be under revision control? Isn't it a local file that NPM uses to acquire sole access to a project?

Reply:
Yeah it's meant to be version controlled